test/release-test.js

```diff
@@ -39,7 +39,7 @@ describe(_release, function() {
     });
   }
 
-  it('works', async function() {
+  it.only('works', async function() {
```

Comment:
This seems like an accident.

Reply:
ah, it sneaked in this commit but overall doesn't show in diff

Reply:
resolved